terraform/services/admin-aco-deny/main.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   full_name   = "${var.app}-${var.env}-admin-aco-deny"
-  db_sg_name  = "bcda-${var.env == "sbx" ? "opensbx" : var.env}-rds"
+  db_sg_name  = "bcda-${var.env == "sandbox" ? "sandbox" : var.env}-rds"
```

Comment:
If this change is made then admin-aco-deny would be unable to connect to the database in legacy.